Formula/portable-openssl.rb

```diff
@@ -3,16 +3,16 @@
 class PortableOpenssl < PortableFormula
   desc "SSL/TLS cryptography library"
   homepage "https://openssl.org/"
-  url "https://www.openssl.org/source/openssl-1.1.1t.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-1.1.1t.tar.gz"
-  mirror "https://www.openssl.org/source/old/1.1.1/openssl-1.1.1t.tar.gz"
-  sha256 "8dee9b24bdb1dcbf0c3d1e9b02fb8f6bf22165e807f45adeb7c9677536859d3b"
-  license "OpenSSL"
+  url "https://www.openssl.org/source/openssl-3.1.2.tar.gz"
```

Comment:
Using 3.0.x, which is an LTS branch, might be easier. Not opposed to always just using the latest 3.z though.

Reply:
Based on the Ruby discussion above, it looks like we'll be updating Portable Ruby around 1st April (+ patch releases where necessary of course). Given that, I think OpenSSL 3.1 should be fine given it's supported until 14th March 2025 (so good for the next two minor versions under our plan). That is only a year before 3.0 EOL and there will be a 3.2+ by then too.

3.1 should just be [3.0 with better performance](https://www.openssl.org/blog/blog/2023/03/07/OpenSSL3.1Release/) really.